studies/models.py

```diff
@@ -940,6 +961,24 @@ def download_url(self):
         return get_download_url(self.full_name)
 
 
+@receiver(pre_delete, sender=Video)
+def delete_s3_video(sender, instance, using, **kwargs):
+    """Kick off tasks to delete S3 videos from the cloud.
+
+    Why are we not overriding delete here? Per the following:
+    https://docs.djangoproject.com/en/1.11/topics/db/models/#overriding-predefined-model-methods
+
+      " Note that the delete() method for an object is not necessarily called when deleting objects in bulk using a
+        QuerySet or as a result of a cascading delete. To ensure customized delete logic gets executed, you can use
+        pre_delete and/or post_delete signals.
+      "
+    """
+    video = instance  # Again, instance is hooked as a kwarg.
+    delete_video_from_cloud.apply_async(
+        video.full_name, countdown=60 * 60 * 24 * 7
+    )  # Delete after 1 week.
```

Comment:
Deleting after 1 week is very sensible, will also want to act directly on researcher permissions for videos so they're not accessible in the interim

Reply:
Since the video models are deleted, there's no real way for the views to know about what's in S3 (i.e. we aren't using `attachment_helpers` anymore)

Reply:
Are there definitely no other cases where we'd want to delete a video model without deleting the actual footage?

Reply:
Accounting for the "not very cautious contributor" case